proxy/src/main/java/com/velocitypowered/proxy/protocol/StateRegistry.java

```diff
@@ -419,11 +450,13 @@ public enum StateRegistry {
           map(0x3D, MINECRAFT_1_19_3, true),
           map(0x41, MINECRAFT_1_19_4, true),
           map(0x43, MINECRAFT_1_20_2, true),
-          map(0x45, MINECRAFT_1_20_3, true));
+          map(0x45, MINECRAFT_1_20_3, true),
+          map(0x46, MINECRAFT_1_20_5, true));
```

Comment:
Shouldn't the respawn packet id be 0x47?

Reply:
good catch!